geopandas/base.py

```diff
@@ -2717,23 +2653,7 @@ def geom_almost_equals(self, other, decimal=6, align=None):
         )
 
     def geom_equals_exact(self, other, tolerance, align=None):
-        """Return True for all geometries that equal aligned *other* to a given
-        tolerance, else False.
-
-        The operation works on a 1-to-1 row-wise manner:
-
-        .. image:: ../../../_static/binary_op-01.svg
-           :align: center
-
-        Parameters
-        ----------
-        other : GeoSeries or geometric object
-            The GeoSeries (elementwise) or geometric object to compare to.
-        tolerance : float
-            Decimal place precision used when testing for approximate equality.
-        align : bool | None (default None)
-            If True, automatically aligns GeoSeries based on their indices.
-            If False, the order of elements is preserved. None defaults to True.
+        """Return for each aligned geometry if it equals *other* to a given tolerance.
```

Comment:
Why are you removing all this? Can you revert this change?

Reply:
Not sure what happened here. Rolled back.